.svnignore

```diff
@@ -3,3 +3,4 @@
 .gitattributes
 .svnignore
 node_modules
+.circlci
```

Comment:
.circleci ?

Reply:
No point in bundling our CircleCI config into the SVN / Plugin repo

Reply:
nvm, I see the typo :laughing: